form.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css" />
+  <script src="form.js" defer></script>
+  <title>Form</title>
+</head>
+<body>
+  <body data-js="body">
+    <header class="header">
+    <h1
+      class="header__headline"
+      >Your Q&A's
+    </h1>
+    </header>
+  <main>
+      <form data-js="form" class="form">
+        <fieldset>
+          <legend>Questions & Answers</legend>
+          <label for="yourQuestionArea">Your Question:</label>
+          <textarea
+            type="text"
+            data-js="questionMessage"
+            name="yourQuestionArea"
+            id="yourQuestionArea"
+            maxlength="150"
+            cols="40"
+            rows="4"
+            placeholder="enter your Question here..."
+            required
+          ></textarea>
+          <small class="form__character-count">
+            (<span data-js="amountLeftQuestion"></span> characters left)</small>
+          <label for="yourAnswerArea">The Answer:</label>
+          <textarea
+            type="text"
+            data-js="answerMessage"
+            name="yourAnswerArea"
+            id="yourAnswerArea"
+            maxlength="150"
+            cols="40"
+            rows="4"
+            placeholder= "read the Answer here..."
+            required
+          ></textarea>
+          <small class="form__character-count">
+            (<span data-js="amountLeftAnswer"></span> characters left)</small>
+          <label for="yourTag">Your Tag:</label>
+          <input
+            type="text"
+            name="yourTag"
+            id="yourTag"
+            placeholder="enter your Tag here..."
+            required
+          ></input>
+        </fieldset>
+        <button class="submitButton">submit</button>
```

Comment:
try to be explicit about the types of your buttons:

```suggestion
        <button class="submitButton" type="submit">submit</button>
```